src/index.html

```diff
@@ -8,9 +8,43 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="searchbar"
+    >
+      <img
+        src="images/Search.svg"
+        alt="searchIcon"
+        class="iconSearch"
+      >
+      <input type="text"
+             class="input"
+             name="searchInput"
+             placeholder="Try ”Los Angeles”"
+             data-qa="big"
+      >
+    </form>
```

Comment:
[point 5 in Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)